src/test/java/be/syntra/devshop/DevshopFront/controllers/SearchControllerTest.java

```diff
@@ -46,203 +42,180 @@
         final CartDto dummyCart = getCartWithMultipleNonArchivedProducts();
-        final SearchModel dummySearchModel = new SearchModel();
-        BigDecimal priceLow = BigDecimal.ZERO;
-        BigDecimal priceHigh = BigDecimal.TEN;
-        dummySearchModel.setDescription(description);
-        dummySearchModel.setPriceLow(priceLow);
-        dummySearchModel.setPriceHigh(priceHigh);
-        dummySearchModel.setSearchRequest(searchRequest);
-
-        when(productListCacheService.findBySearchRequest(any())).thenReturn(dummyProductList);
-        when(productListCacheService.sortListByName(any(), any())).thenReturn(dummyProductList);
-        when(productListCacheService.sortListByPrice(any(), any())).thenReturn(dummyProductList);
-        when(productListCacheService.filterByPrice(dummyProducts, dummySearchModel)).thenReturn(dummyProductList);
-        when(productListCacheService.searchForProductDescription(dummyProducts, dummySearchModel)).thenReturn(dummyProductList);
+        when(productService.findAllProductsBySearchModel()).thenReturn(dummyProductListDto);
+        when(productMapperUtil.convertToProductDtoList(any())).thenReturn(getDummyProductDtoList());
+        when(searchService.getSearchModel()).thenReturn(new SearchModel());
         when(cartService.getCart()).thenReturn(dummyCart);
-        when(searchService.getSearchModel()).thenReturn(dummySearchModel);
 
         // when
-        final ResultActions getResult = mockMvc.perform(get(url));
+        final ResultActions getResult = mockMvc.perform(get("/search/pricehigh/?priceHigh=" + PRICE_HIGH));
 
         // then
         getResult
                 .andExpect(status().isOk())
                 .andExpect(view().name("product/productOverview"))
                 .andExpect(content().contentType("text/html;charset=UTF-8"))
-                .andExpect(model().attributeExists("products"))
-                .andExpect(model().attribute("products", dummyProducts))
-                .andExpect(model().attribute("cart", dummyCart))
-                .andExpect(model().attribute("searchModel", dummySearchModel));
+                .andExpect(model().attributeExists("searchModel", "productlist", "cart"));
 
-        verify(productListCacheService, times(1)).findBySearchRequest(any());
-        verify(productListCacheService, times(1)).filterByPrice(any(), any());
+        verify(searchService, times(1)).setPriceHigh(price_low_big_d);
+        verify(searchService, times(1)).setSearchResultView(true);
+        verify(searchService, times(1)).setArchivedView(false);
+        verify(productService, times(1)).findAllProductsBySearchModel();
     }
 
-    @ParameterizedTest
-    @ValueSource(strings = {"/search/archived/sortbyname", "/search/archived/sortbyprice"})
-    public void canSortArchivedProductsTest(String url) throws Exception {
+    @Test
+    void searchProductDescriptionTest() throws Exception {
+        //given
+        final String DESCRIPTION = "my prod description";
```

Comment:
not a static String. Convention-wise, it shouldn't be allCaps

Reply:
idd, changed